Fix missing links in INSTALL.rst and simplify language (#6984)
scikit-image/scikit-image: INSTALL.rst

How you should install ``scikit-image`` depends on your needs and skills:

- First, ensure that you have Python installed.

>   Two popular alternatives are the pip-based
>   `Python.org installers <https://www.python.org/downloads/>`_
>   and the conda-based
>   `miniforge <https://github.com/conda-forge/miniforge>`_.

Reviewer: ```suggestion
  Two popular alternatives are
  `Python.org installers <https://www.python.org/downloads/>`_ which provide pip and
  `miniforge <https://github.com/conda-forge/miniforge>`_ which provides conda.
```
This sounds "more correct" and clearer to me.

Reviewer 2: Hmm, I prefer the original.

Reviewer 2: Hm, I believe that you understand 'based' in the strictest sense. Maybe think of a 'web-based application?' It doesn't mean that the application would be made out of the web, from the web, nor with the web as its main ingredient... Rather, it means that it uses the web; stretching it a bit much (in the broadest/weakest sense), it just means that it pertains to the web.

Reviewer: Hmm, my gut-feeling is unchanged, but this is not a blocker for me anyway. :wink:

Author: I'm setting the stage here for the next instruction, which requires users clearly knowing which of the two installers they have available. Sometimes, instructions are clearer even when they are not technically 100% accurate :)

Reviewer 3: @lagru It looks like this is still an open issue. I am going to go ahead merge it anyway, since it is a good improvement already. We should improve this further in a subsequent PR.
